modules/ROOT/pages/permissions-by-product.adoc

```diff
@@ -10,9 +10,11 @@ The following permissions can be assigned to xref:teams.adoc[teams], granted to
 Organization Administrator:: At the root organization level, grants a user most permissions available in Anypoint Platform, including but not limited to secrets management, network administration, and other view, modify, execute, and delete permissions. The Organization Administrator permission also grants access to the Organization Administration page, where the user can add and manage users and permissions, view and edit organization details, access API Manager > Client Applications, access the client ID and client secret for the organization, and customize the theme of the Developer Portal. This permission enables a user to edit all versions of all APIs, all registered applications, and all API Portals in Anypoint Platform. +
 For security reasons, MuleSoft recommends distributing this permission to as few users as possible.
 
+Audit Log Config Manager:: Enables a user to configure the retention period for audit logs over their organization. Note that you must apply this permission at the root organization level.
```

Comment:
This can be set/viewed from new UI only, should we call that out.

Reply:
```suggestion
Audit Log Config Manager:: Enables a user to configure the retention period for audit logs over their organization. Note that you must apply this permission at the root organization level. This permission appears only if you have *Try new features* or the Teams feature enabled in Access Management.
```